src/theme/theme.ts

```diff
@@ -37,31 +38,36 @@ const getDesignTokens = (mode: PaletteMode) => ({
       fontSize: '26px',
       fontFamily: 'monospace',
       lineHeight: "38px",
-      fontWeight: "bold"
+      fontWeight: "bold",
+      WhiteSpace: 'nowrap'
```

Comment:
just to remember, the reason I switched from whiteSpace ==> WhiteSpace is that typescript was yelling at me!